crates/steel-core/src/primitives/strings.rs

```diff
@@ -11,8 +11,6 @@ fn char_upcase(c: char) -> char {
     c.to_ascii_uppercase()
 }
 
-/// # steel/strings
-///
```

Comment:
I feel like these titles are unnecessary, since the the doc generator knows the module name already. In the book you can actually see two consecutive`steel/strings` titles.